Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/ml_controller.hpp

```diff
@@ -64,33 +58,17 @@ class MLController {
 				results_.push_back(model->do_work(color_matrix, depth_matrix));
 			}
 		);
-
-		//tbb::parallel_for_each(
-		//	rgb_models.begin(),
-		//	rgb_models.end(),
-		//	[&] (ModelInterface<rs2::video_frame>* model)
-		//	{
-		//		std::cout << "Doing parallel rgb foreach\n";
-		//		results_.push_back(model->do_work(frameset.get_color_frame()));
-		//	}
-		//);
-		//
-		//tbb::parallel_for_each(
-		//	depth_models.begin(),
-		//	depth_models.end(),
-		//	[&](ModelInterface<rs2::depth_frame>* model)
-		//	{
-		//		std::cout << "Doing parallel depth foreach\n";
-		//		results_.push_back(model->do_work(frameset.get_depth_frame()));
-		//	}
-		//);
 	}
-
+	/// <summary>
+	/// Create a new object recognition model and add it to the list containing current object recognition models
+	/// </summary>
+	/// <param name="model">new object recognition model created</param>
 	void add_model(ModelInterface& model)
 	{
 		models_.push_back(&model);
 	}
 
+	// The following two functions are for testing and demonstration, not used in the actual system
```

Comment:
Remove this and the commented out code.
Its generally also not good form to include testing and demonstration code as commented out 😄  They should probably be tests or not included